wdl/tasks/AlignReads.wdl

```diff
@@ -104,16 +104,16 @@ task SAMtoPAF {
         filename=$(basename -- ~{sam_formatted_file})
         extension="${filename##*.}"
         if [[ "$extension" == "sam" ]]; then
-            /minimap2-2.17_x64-linux/k8 \
-                /minimap2-2.17_x64-linux/paftools.js \
+            /minimap2-2.24_x64-linux/k8 \
```

Comment:
let's use a variable/macro for that here as well.

Reply:
👍